lib/nerves_network/utils.ex

```diff
@@ -26,6 +28,47 @@ defmodule Nerves.Network.Utils do
     "169.254.#{x}.#{y}"
   end
 
+  defp bits_set(128), do: 1
```

Comment:
you can probably make this concise with using a matching clause, like:
`defp bits_set(bits) when bits in [1, 2, 4, 8, 16, 32, 64, 128], do: 1`

Reply:
done